node_modules/oae-core/deleteresource/deleteresource.html

```diff
@@ -20,6 +20,8 @@ <h3 id="deleteresource-modal-title">
                             __MSG__DELETE_FILE__
                         {elseif contextProfile.resourceSubType === 'link'}
                             __MSG__DELETE_LINK__
+                        {elseif contextProfile.resourceSubType === 'doodle'}
```

Comment:
Are resourceSubTypes `doodle` and `loodle` different? They seem to be used interchangeably.